components/tree-view/padding.ts

```diff
@@ -12,21 +11,21 @@ import { Directive, Input } from '@angular/core';
   providers: [{ provide: CdkTreeNodePadding, useExisting: NzTreeNodePaddingDirective }]
 })
 export class NzTreeNodePaddingDirective<T> extends CdkTreeNodePadding<T> {
-  _indent = 24;
+  override _indent = 24;
 
   @Input('nzTreeNodePadding')
-  get level(): number {
+  override get level(): number {
     return this._level;
   }
-  set level(value: NumberInput) {
+  override set level(value: number) {
```

Comment:
https://github.com/angular/components/blob/master/src/cdk/tree/padding.ts#L41
if type equals `number` instead of `NumberInput` like `cdk`, it may throw invalid type errors when using like this:
```typescript
<nz-tree-node *nzTreeNodeDef="let node" nzTreeNodePadding>
...
</nz-tree-node>
```

Reply:
Reverted.